File: 0476-number-complement/0476-number-complement.java
class Solution {
    public int findComplement(int num) {
         int ans = 0;
        int i = 0;
        while(num > 0){
            int x = num%2;
            if(x == 0) x = 1;
            else if(x == 1) x = 0;
            ans = ans + (int)(Math.pow(2,i) * x);
            num /=2;
            i++;

        }
        return ans;
    }
}

File: 0476-number-complement/0476-number-complement.py
class Solution(object):
    def findComplement(self, num):
        ans = 0
        i = 0
        while(num > 0):
            x = num%2
            if(x == 1): x = 0
            elif(x == 0): x = 1
            ans = ans + pow(2,i)*x
            i+=1
            num/=2
        
        return ans



        """
        :type num: int
        :rtype: int
        """
        